sms_notifservice.py

```diff
@@ -1,24 +1,43 @@
 import boto3
+from onevizion import NotificationService, LogLevel
 
 from curl import Curl
-from integration_log import LogLevel
-from notifservice import NotificationService
+
+HTTPS = "https://"
+HTTP = "http://"
+
+
+def get_scheme_url(url):
+    if url.startswith(HTTPS) or url.startswith(HTTP):
+        return url
+    else:
+        return HTTPS + url
+
+
+def get_url_without_scheme(url):
+    if url.startswith(HTTPS) or url.startswith(HTTP):
+        return url.replace(HTTPS, "").replace(HTTP, "")
+    else:
+        return url
 
 
 class SmsNotifService(NotificationService):
 
-    def __init__(self, service_id, process_id, ov_url, ov_username, ov_pwd, phone_number_field_name, access_key_id,
-                 secret_access_key, aws_region, max_attempts=1, next_attempt_delay=30):
-        super().__init__(service_id, process_id, ov_url, ov_username, ov_pwd, max_attempts, next_attempt_delay)
+    def __init__(self, service_id, ov_url, ov_username, ov_pwd, phone_number_field_name, access_key_id,
```

Comment:
Not quite on the topic of the issue, but I think it needs to be discussed: 
1. It confuses me that there is no data(from settings.json) validation in the current integration. Create an issue for adding this?
2. We pass a lot of parameters, some of them are the data that is needed for authentication. Is it worth combining them, for example, like here (https://github.com/ov-integrations/ov-monitoring/blob/master/auth_data.py)?

Reply:
1 yes
2 no

Reply:
[IHub-186674](https://trackor.onevizion.com/form/ConfigAppForm.do?id=100104477284&ttid=10009161)